vrs/marten-construction-script.py

```diff
@@ -0,0 +1,168 @@
+ht_10k_indel = ht_10k_indel.checkpoint(indel_path,overwrite = True)
+
+# 10,000 variants with long reference alleles
+ht_10k_longRef = ht_whole.filter(hl.len(ht_whole.alleles[0])>5)
+longRef_sample_freq = 10000/(whole_count*ref_freq)
+ht_10k_longRef = ht_10k_longRef.sample(longRef_sample_freq).repartition(10)
+# TO ADD: ht_10k_longRef = ht_10k_longRef.repartition(100)
+longRef_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_longRef_temp.ht'
+ht_10k_longRef = ht_10k_longRef.checkpoint(longRef_path,overwrite = True)
+
+# 10,000 variants with long variant alleles
+ht_10k_longVar = ht_whole.filter(hl.len(ht_whole.alleles[1])>5)
+longVar_sample_freq = 10000/(whole_count*var_freq)
+ht_10k_longVar = ht_10k_longVar.sample(longVar_sample_freq).repartition(100)
+# TO ADD: ht_10k_longVar = ht_10k_longVar.repartition(100)
+longVar_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_10k_longVar_temp.ht'
+ht_10k_longVar = ht_10k_longVar.checkpoint(longVar_path,overwrite = True)
+
+# 9,000 X-Chromosome Variants
+ht_9k_x = ht_whole.filter(ht_whole.locus.contig == "chrX")
+x_sample_freq = 9000/(whole_count*x_freq)
+ht_9k_x = ht_9k_x.sample(x_sample_freq).repartition(100)
+# TO ADD: ht_9k_x = ht_9k_x.repartition(100)
+xchr_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_9k_x_temp.ht'
+ht_9k_x = ht_9k_x.checkpoint(xchr_path,overwrite = True)
+
+# 1,000 Y-Chromosome Variants 
+ht_1k_y = ht_whole.filter(ht_whole.locus.contig == "chrY")
+y_sample_freq = 1000/(whole_count*y_freq)
+ht_1k_y = ht_1k_y.sample(y_sample_freq)
+ht_1k_y = ht_1k_y.repartition(100)
+ychr_path = 'gs://gnomad-tmp-4day/marten-temp-01-06/ht_1k_y_temp.ht'
+ht_1k_y = ht_1k_y.checkpoint(ychr_path,overwrite = True)
+
+# Combine separate tables into one 
+
+tableParts = [ht_50k, ht_10k_indel, ht_10k_longRef, ht_10k_longVar, ht_9k_x, ht_1k_y]
+ht_union = tableParts[0]
+for item_to_union in tableParts[1:]:
+    ht_union = ht_union.union(item_to_union) 
+    # final table of all but the multiallelic sites
+    # at this point, approx ~90,000 variants
+
+# cannot figure how to make LOGGER work ? 
+print(f'Size of ~50k random, ~10k indel, ~10k long ref, ~10k long var, ~10k sex chr as: {ht_union.count()}')
+
+# Construct a table of only multiallelic sites
+
+# for 3 random partitions, only takes the highly multiallelic sites (>5 variants per loci)
+ht_filt = ht_whole._filter_partitions(sample(range(ht_whole.n_partitions()), 3))
+group_ht = ht_filt.group_by(ht_filt.locus).aggregate(n=hl.agg.count())
+group_ht = group_ht.filter(group_ht.n > 5)
+ 
+multiAlleleCount = group_ht.count()
+print(f'There are {multiAlleleCount} very multiallelic SITES in the table above') # 585 sites, plenty of variables
+ 
+# collects all variants at highly multiallelic loci    
+multiList = group_ht.locus.collect(_localize = False)
+ht_multiallelic = ht_filt.filter(multiList.contains(ht_filt.locus))
+# returns 9602 variants on one trial
+
+# checkpoint for final multiallelic variants
```

Comment:
```suggestion
```
I'm not sure you need this comment for this block (though it was good to err on the side of overdocumentation)